go.mod

```diff
@@ -8,10 +8,10 @@ require (
 	github.com/onsi/gomega v1.19.0
 	github.com/openshift/api v3.9.0+incompatible
 	github.com/xanzy/go-gitlab v0.68.2
-	k8s.io/api v0.24.2
-	k8s.io/apiextensions-apiserver v0.24.2
-	k8s.io/apimachinery v0.24.2
-	k8s.io/client-go v0.24.2
+	k8s.io/api v0.24.3
```

Comment:
cool a good cross reference with the older versions application-service is bringing in :-)